Src/Generic/Extensions/UIImageView+Cache.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  UIImageView+Cache.swift
+//  Hedvig
+//
+//  Created by Gustaf Gunér on 2019-03-21.
+//  Copyright © 2019 Hedvig AB. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Disk
+
+extension UIImageView {
+    func imageFromURL(url: String) -> Void {
```

Comment:
Not too sure if I like this, due to it being so hidden from the consumer (i.e a hidden function for initializing)
would prefer a viewable called `CachedImage` I think.

Reply:
Good point!